python/llm/portable-zip/chat.py

```diff
@@ -276,31 +321,43 @@ def auto_select_model(model_name):
     parser = argparse.ArgumentParser()
     parser.add_argument("--model-path", type=str, help="path to an llm")
     parser.add_argument("--start-size", type=int, default=4, help="start_size of kv_cahce")
+    parser.add_argument("--recent-size", type=int, default=2000)
     args = parser.parse_args()
 
     model_path = args.model_path
     start_size = args.start_size
+    recent_size = args.recent_size
 
     model = auto_select_model(model_path)
     model = optimize_model(model)
 
     tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
+    print("model.config.architectures[0] is: ", model.config.architectures[0])
```

Comment:
I feel these print messages could be removed?

Reply:
fixed